public/style.css

```diff
@@ -0,0 +1,8 @@
+* { margin: 0; padding: 0; box-sizing: border-box; }
```

Comment:
Jesus ahi! why inlining?!

Reply:
Copy Pasting brother

Reply:
They said in class that our client side won't matter .. it's node excersice man